app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OrganizationsController < ApplicationController
+  before_action :redirect_to_root, unless: :logged_in?
+
+  before_action :verify_organization_ownership, except: [:new, :create]
+  before_action :set_organization,              except: [:new, :create]
+
+  def new
+    @organization               = Organization.new
+    @users_github_organizations = current_user.github_client.users_organizations.
+                                  collect { |org| [org.login, [org.login, org.id]] }
+  end
+
+  def create
+    org               = JSON.parse(params[:org])
+    login, github_id  = org.first, org.last
+
+    if Organization.where(github_id: github_id).present?
```

Comment:
There might be an edge case here. What happens if two users have added the same organization as a classroom?

I can think of two ways to handle this:

1. Any user on the owners team can access the classroom using the classroom app
2. Scope classrooms to users

I think the first option makes the most sense.